### PR TITLE
Prepare auto-pts for EATT testing

### PR DESCRIPTION
**Pull request by KKopyscinski**
-EATT test cases are as follows:
	1. GATT/SR/GAC/BI-01-C
	2. GATT/CL/GAR/BV-11-C
	3. GATT/SR/GAR/BV-12-C
	4. GATT/CL/GAS/BV-04-C
	5. GATT/SR/GAR/BV-11-C
	6. GATT/CL/GAR/BV-10-C
	7. GATT/SR/GAW/BV-12-C
	8. GATT/CL/GAW/BI-37-C
	9. GATT/SR/GAW/BV-13-C
	10. GATT/SR/GAW/BV-14-C
	11. GATT/CL/GAW/BV-10-C
	12. GATT/SR/GPM/BV-05-C
-Edited workspace to enable these tests (for GATT/SR/GAC/BI-01-C TSPC_GATT_2_3 should be disabled seperately)
-Added appropriate wids
-corrected hdl_wid_76
-for test 7-12 PTS should receive errata - they require more than one EATT channel, which, according to specification,  shouldn't be required

### CODE REVIEW

**Comment by Reynevan94** [comment]
PASS:
-------------------------------------------------------------------------------
GATT/SR/GAC/BI-01-C
GATT/SR/GAR/BV-11-C
GATT/SR/GAW/BV-13-C
GATT/CL/GAS/BV-04-C
GATT/CL/GAW/BI-37-C

FAIL/INCONCLUSIVE and should pass:
-------------------------------------------------------------------------------
GATT/CL/GAR/BV-10-C
GATT/SR/GPM/BV-05-C

MyNewt doesn't support, but are not required:
-------------------------------------------------------------------------------
GATT/CL/GAR/BV-11-C
GATT/SR/GAR/BV-12-C

Not passing, but included in errata no. 13112 - haven't been run with changed ICSs
-------------------------------------------------------------------------------
GATT/SR/GAW/BV-12-C
GATT/CL/GAW/BV-10-C
GATT/SR/GAW/BV-14-C

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (michal-narajowski) [inline .idea/.gitignore:1]: Remove .idea/ from changes

File: workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6 [hunk 2]
```diff
@@ -1924,7 +1924,7 @@
           <Row>
             <Name>TSPC_GATT_1a_2</Name>
             <Description>GATT Client over BR/EDR (C.2)</Description>
-            <Value>FALSE</Value>
+            <Value>TRUE</Value>
```
Comment (michal-narajowski) [inline workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6:1927]: Not sure how this change happen but we do not support BR/EDR, so this probably should be FALSE.

File: workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6 [hunk 3]
```diff
@@ -2429,7 +2429,7 @@
             <Name>TSPX_bd_addr_iut</Name>
             <Description>The unique 48-bit Bluetooth device address (BD_ADDR) of the IUT. This was filled in during workspace creation.</Description>
             <Type>OCTETSTRING</Type>
-            <Value>000000000000</Value>
+            <Value>112233445566</Value>
```
Comment (michal-narajowski) [inline workspaces/Mynewt Nimble Host/Mynewt Nimble Host.pqw6:2432]: This change is not really necessary.

File: ptsprojects/mynewt/gatt_wid.py [hunk 4]
```diff
@@ -1256,6 +1259,44 @@ def hdl_wid_122(desc):
...
+    return True
+
+def hdl_wid_400(desc):
+    log("This should be handled by Mynewt automatically")
+    return '0000'
+
+
+def hdl_wid_402(desc):
+    log("This should be handled by Mynewt automatically")
+
```
Comment (michal-narajowski) [inline ptsprojects/mynewt/gatt_wid.py:1296]: Superfluous line.

**Comment by KKopyscinski** [comment]
Test summary:
PASSING:
-GATT/SR/GAC/BI-01-C
-GATT/SR/GAR/BV-11-C
-GATT/SR/GAW/BV-13-C
-GATT/CL/GAS/BV-04-C
-GATT/CL/GAW/BI-37-C - approved erratum will remove this test
-GATT/CL/GAR/BV-10-C
-GATT/CL/GAW/BV-10-C- approved erratum will remove this test

FAILING MANDATORY:
-GATT/SR/GPM/BV-05-C - issue should be solved by erratum 14993
-GATT/SR/GAW/BV-12-C - filed erratum 13112
-GATT/SR/GAW/BV-14-C - filed erratum 13112

UNSUPPORTED NON-MANDATORY:
-GATT/CL/GAR/BV-11-C
-GATT/SR/GAR/BV-12-C

File: ptsprojects/mynewt/post_tc.py [hunk 5]
```diff
@@ -23,7 +23,7 @@
 
 CONFIG_PROC = None
 # XXX: Fill me - nrfjprog path example: /home/user/tool/nrfjprog
-CONFIG_PATH = None
+CONFIG_PATH = '/usr/bin/nrfjprog'
```
Comment (MariuszSkamra) [inline ptsprojects/mynewt/post_tc.py:26]: I guess it's not in the scope of this PR

File: pybtp/btp.py [hunk 6]
```diff
@@ -185,6 +185,8 @@
                   CONTROLLER_INDEX),
     "read_multiple": (defs.BTP_SERVICE_ID_GATT, defs.GATT_READ_MULTIPLE,
                       CONTROLLER_INDEX),
+    "read_multiple_var": (defs.BTP_SERVICE_ID_GATT, defs.GATT_READ_MULTIPLE_VAR,
+                      CONTROLLER_INDEX),
```
Comment (MariuszSkamra) [inline pybtp/btp.py:189]: align

Comment (KKopyscinski) [reply pybtp/btp.py:189]: done

File: ptsprojects/mynewt/post_tc.py (hunk 5 above)
Comment (KKopyscinski) [reply ptsprojects/mynewt/post_tc.py:26]: Sure, it isn't :)

**Pull request merged**